### DISCUSSION TITLE
How to fill in the values of 'NEXTAUTH_SECRET' and 'DATABASEURL' for Vercel deployment

### DISCUSSION THREAD

**Discussion by jellzone** [reworkd/AgentGPT#824] [Q&A] [answered]
How to fill in the values of 'NEXTAUTH_SECRET' and 'DATABASEURL' for Vercel deployment

**Comment by jasangill1** [comment, marked as answer]
Hey man the NEXTAUTH_SECRET is just a random series of characters that you can create it can be as simple as 123415 or something as complex as dawydahwd21231e1eyh typically people use websites like this https://www.random.org/strings/ too generate unique keys for NEXTAUTH_SECRET. for the database URL you might need to set up an account with the database company and get a URL that you can use. Let me know if this helps 😎
